tests/unit/Model/Command/UserRecommendationTest.php

```diff
@@ -10,18 +10,15 @@ class UserRecommendationTest extends TestCase
     /** @test */
     public function shouldBeInstantiableViaNamedConstructorWithDefaultValues(): void
     {
-        $command = UserRecommendation::create('user-id', 333, 'test-scenario', 1.0, 3600);
+        $command = UserRecommendation::create('user-id', 'test-scenario');
 
         $this->assertInstanceOf(UserRecommendation::class, $command);
-        $this->assertSame(
+        $this->assertEquals(
             [
                 'type' => 'user-based-recommendations',
                 'parameters' => [
                     'user_id' => 'user-id',
-                    'count' => 333,
                     'scenario' => 'test-scenario',
-                    'rotation_rate' => 1.0,
-                    'rotation_time' => 3600,
                     'hard_rotation' => false,
                     'min_relevance' => MinimalRelevance::LOW,
```

Comment:
I may be wrong, but I expected only user_id and scenario in the request, as these are the only parameters set for the instance of UserRecommendation. Why are there some defaults?

Reply:
You are correct. They're here because I goofed